package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "vtex-tachyons",
-  "version": "2.8.0",
+  "version": "2.8.3",
   "description": "A custom Tachyons for VTEX",
   "main": "tachyons.css",
   "scripts": {
-    "prepublishOnly": "node index.js"
+    "prepublishOnly": "node index.js",
+    "postreleasy": "npm publish"
```

Comment:
`releasy --npm` resolveria isso

poderia fazer aquele "releasy config" malandro lá

Reply:
Não to ligado nisso